UnityProject/Assets/Scripts/Core/Physics/UniversalObjectPhysics.PushPull.cs

```diff
@@ -311,8 +311,9 @@ public void ForceTilePush(Vector2Int worldDirection, List<Physics.UniversalObjec
 				var inDirection = cachedPosition - Pulling.Component.transform.position;
 				if (inDirection.magnitude > 2f && isServer)
 				{
+					var InPulling = Pulling.Component;
 					PullSet(null, false, ResetForcePosition: true);
-					Pulling.Component.ResetLocationOnClients(true);
+					InPulling?.ResetLocationOnClients(true);
```

Comment:
Why are you delcaring InPulling for only one line?
Wouldn't it be betterr to go:

Pulling.Component?.ResetLocationOnClients(true);